source/amazon/services/troubleshooting.rst

```diff
@@ -8,46 +8,232 @@ Troubleshooting
+
+The module does not work and the following error messages appears in the ``ossec.log``:
 
     .. code-block:: none
         :class: output
 
-        2019/10/28 13:58:11 wazuh-modulesd:aws-s3: WARNING: Bucket: wazuh-cloudtrail  -  Returned exit code 3
-        2019/10/28 13:58:11 wazuh-modulesd:aws-s3: WARNING: Bucket: wazuh-cloudtrail  -  Invalid credentials to access S3 Bucket
+        2022/03/03 16:06:56 wazuh-modulesd:aws-s3: WARNING: Bucket:  -  Returned exit code 3
+        2022/03/03 16:06:56 wazuh-modulesd:aws-s3: WARNING: Bucket:  -  Invalid credentials to access S3 Bucket
+
+**Solution**
+
+Make sure the credentials provided grant access to the requested S3 bucket and the bucket itself exists in the given region, if specified.
+
+
+The config profile could not be found
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The module does not work and the following error messages appears in the ``ossec.log``:
+
+    .. code-block:: none
+        :class: output
+
+        2022/03/03 15:49:34 wazuh-modulesd:aws-s3: WARNING: Bucket:  -  Returned exit code 12
+        2022/03/03 15:49:34 wazuh-modulesd:aws-s3: WARNING: Bucket:  -  The config profile (default) could not be found
+
+**Solution**
+
+Make sure the profile value specified in the configuration matches with an existing one placed in the ``~/.aws/credentials`` file. Check the :ref:`Configuring AWS credentials <aws_profile>` section to learn more about how to configure a profile for authentication.
+
 
-    The exit codes and their possible remediations are the next:
+The security token included in the request is invalid
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+The module does not work and the following error messages appears in the ``ossec.log``:
+
+    .. code-block:: none
+        :class: output
+
+        2022/03/03 16:16:18 wazuh-modulesd:aws-s3: WARNING: Service: cloudwatchlogs  -  Returned exit code 12
+        2022/03/03 16:16:18 wazuh-modulesd:aws-s3: WARNING: Service: cloudwatchlogs  -  An error occurred (InvalidClientTokenId) when calling the GetCallerIdentity operation: The security token included in the request is invalid.
+
+**Solution**
+
+No credentials were provided to attempt to access to CloudWatch Logs or that the credentials provided don't grant access to CloudWatch Logs. Check the :ref:`Configuring AWS credentials <amazon_credentials>` section to learn more about the different options available and how to configure them.
+
+
+There are no AWS alerts present in the Wazuh UI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The module is running but no alerts are displayed in the Wazuh UI.
+
+**Solution**
+
+First of all, make sure there is no ``ERROR`` or ``WARNING`` messages in the ``ossec.log`` file by :ref:`enabling debug mode <aws_debug_mode>`. If the module is running as expected but no alerts are being generated it could mean there is no reason for alerts to be raised in first place. Check the following to verify this:
```

Comment:
```suggestion
First of all, make sure there are no ``ERROR`` or ``WARNING`` messages in the ``ossec.log`` file by :ref:`enabling debug mode <aws_debug_mode>`. If the module is running as expected but no alerts are being generated it could mean there is no reason for alerts to be raised in first place. Check the following to verify this:
```

Reply:
Fixed